app/Components/NavBar.tsx

```diff
@@ -0,0 +1,39 @@
+import Image from "next/image";
+import Link from "next/link";
+
+export default function NavBar() {
+  // return <p>test</p>;
+
+  return (
+    <div className="flex justify-between items-center font-secondary border-b-2 border-white p-4">
+      <Link href="/">
+        <svg
```

Comment:
Is the idea to have one icon component with multiple icons defined in one component? Then depending on the type, it will determine which icon it should be?